pulsar-broker/src/main/java/org/apache/pulsar/compaction/CompactorMXBean.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.compaction;
+
+import org.apache.bookkeeper.common.annotation.InterfaceAudience;
+import org.apache.bookkeeper.common.annotation.InterfaceStability;
+
+/**
+ * JMX Bean interface for Compactor stats.
+ */
+@InterfaceAudience.LimitedPrivate
+@InterfaceStability.Stable
+public interface CompactorMXBean {
+
+    /**
+     * @return the removed event count of last compact
+     */
+    long getLastCompactRemovedEventCount(String topic);
+
+    /**
+     * @return the timestamp of last succeed compact
+     */
+    long getLastCompactSucceedTimestamp(String topic);
+
+    /**
+     * @return the timestamp of last failed compact
+     */
+    long getLastCompactFailedTimestamp(String topic);
+
+    /**
+     * @return the duration time of last compact
+     */
+    long getLastCompactDurationTime(String topic);
```

Comment:
```suggestion
    long getLastCompactDurationTimeInMills(String topic);
```

I think milliseconds is enough here?

Reply:
Yes, I think so. Using nanoTime to record the start, and convert to mills in addCompactEndOp.
BTW, I will change the method to the above name.